components/widgets/menu/menu_items/submenu_item.test.tsx

```diff
@@ -20,7 +22,15 @@ describe('components/widgets/menu/menu_items/submenu_item', () => {
                 root={true}
             />,
         );
-        expect(wrapper.html()).toEqual('<li class="SubMenuItem MenuItem" role="menuitem" id="1_menuitem"><div id="1" tabindex="0"><span id="channelHeaderDropdownIconLeft_1" class="fa fa-angle-left SubMenu__icon-left-empty" aria-label="submenu icon"></span>test<span id="channelHeaderDropdownIconRight_1" class="fa fa-angle-right SubMenu__icon-right-empty" aria-label="submenu icon"></span><ul class="a11y__popup Menu dropdown-menu SubMenu" style="visibility: hidden;"></ul></div></li>');
+
+        expect(wrapper.html()).toEqual(format(`<li class="SubMenuItem MenuItem" role="menuitem" id="1_menuitem">
```

Comment:
Not sure why we use `wrapper.html()` here, but I definitely appreciate the helper to make it easier to read. Are there any changes here beyond being easier to read?

Reply:
There aren't. Previously I decided to hide the icons completely by setting 'display: block' but that ended up confusing with the styles even more.. so now I just left the helper :P

Reply:
I was thinking to open a ticket to refactor this into a proper snapshot file?

Reply:
That would be a good idea unless there's some reason this wasn't done as a snapshot in the first place. I know the menu components do some stuff that's not React-friendly, so perhaps that's why snapshots weren't used here before.